daemon/container.go

```diff
@@ -54,8 +54,8 @@ func (daemon *Daemon) GetContainer(prefixOrName string) (*container.Container, e
 	return daemon.containers.Get(containerID), nil
 }
 
-// CheckContainer make sure the specified container validates the specified conditions
-func (daemon *Daemon) CheckContainer(container *container.Container, conditions ...func(*container.Container) error) error {
+// checkContainer make sure the specified container validates the specified conditions
+func (daemon *Daemon) checkContainer(container *container.Container, conditions ...func(*container.Container) error) error {
```

Comment:
These checkContainer changes are in the wrong commit; should be in f5c490ac633c25ef5f83e99e524a954d50801fb5